presto-hudi/pom.xml

```diff
@@ -263,6 +264,20 @@
                             <ignoredClassPattern>module-info</ignoredClassPattern>
                             <ignoredClassPattern>META-INF.versions.9.module-info</ignoredClassPattern>
                         </ignoredClassPatterns>
+                        <ignoredDependencies>
+                            <dependency>
+                                <groupId>io.airlift</groupId>
+                                <artifactId>aircompressor</artifactId>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.apache.hudi</groupId>
+                                <artifactId>hudi-presto-bundle</artifactId>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.apache.hive</groupId>
+                                <artifactId>hive-storage-api</artifactId>
+                            </dependency>
+                        </ignoredDependencies>
```

Comment:
@vamsikarnika, the reason I asked is that I’m concerned that simply silencing these errors could cause runtime issues or unexpected behavior if the duplicate classes are from different versions, as we can’t control which version gets loaded at runtime. Do we know if the duplicate classes are actually identical?

> Longer term, we want bring this up with the Hudi team to see if shading those dependencies inside hudi-presto-bundle is possible.

Could you create an issue in Hudi and link this PR to it? If not immediately, we should plan to fix the shaded JAR so that it no longer triggers duplicate class errors.